src/include/block.h

```diff
@@ -315,6 +318,19 @@ struct __wt_block_desc {
  */
 #define WT_BLOCK_DESC_SIZE 16
 
+#define WT_TIERED_TOP_OBJECT_ID 0xFFFFFFFFFFFFFFFFULL
```

Comment:
I find `TOP` a confusing term. Maybe `WT_TIERED_OBJECT_ID_MAX`? Or can this just be defined as `UINT64_MAX` - does it really need its own define or could we just use one of the existing ones?

Reply:
I named this `WT_TIERED_CURRENT_ID` and added a comment, it's a way in the opener API to reference the current id.